src/actions/account.js

```diff
@@ -164,16 +187,12 @@ export const { initializeRecoveryMethod, validateSecurityCode, initTwoFactor, re
         wallet.twoFactor.reInitTwoFactor.bind(wallet.twoFactor),
         () => defaultCodesFor('account.reInitTwoFactor')
     ],
-    SEND_TWO_FACTOR: [
-        wallet.twoFactor.sendRequest.bind(wallet.twoFactor),
-        () => defaultCodesFor('account.sendTwoFactor')
-    ],
     RESEND_TWO_FACTOR: [
         wallet.twoFactor.resend.bind(wallet.twoFactor),
         () => defaultCodesFor('account.resendTwoFactor')
     ],
     VERIFY_TWO_FACTOR: [
-        wallet.twoFactor.verifyTwoFactor.bind(wallet.twoFactor),
+        (...args) => wallet.twoFactor.verifyRequestCode(...args),
```

Comment:
why these changes?

Reply:
@vgrichina disregard this comment. AFAIK the default way to call helper instances based on wallet should be done this way:
```
(...args) => wallet.twoFactor.verifyRequestCode(...args)
```

I forgot the convention and led to lots of confusion re: 2fa refactor.

Reply:
these are your changes, not mine I think :)